stacks/java-quarkus/1.3.1/devfile.yaml

```diff
@@ -0,0 +1,72 @@
+schemaVersion: 2.2.0
+metadata:
+  name: java-quarkus
+  displayName: Quarkus Java
+  description: Java application using Quarkus and OpenJDK 17
+  icon: https://design.jboss.org/quarkus/logo/final/SVG/quarkus_icon_rgb_default.svg
+  tags:
+    - Java
+    - Quarkus
+  projectType: Quarkus
+  language: Java
+  version: 1.3.1
+  website: https://quarkus.io
+starterProjects:
+  - name: community
+    zip:
+      location: https://code.quarkus.io/d?e=io.quarkus%3Aquarkus-resteasy&e=io.quarkus%3Aquarkus-micrometer&e=io.quarkus%3Aquarkus-smallrye-health&e=io.quarkus%3Aquarkus-openshift&cn=devfile
```

Comment:
@michael-valdron 
Can we change the project source to the git repository of the [java quarkus sample](https://github.com/devfile-samples/devfile-sample-code-with-quarkus)? That would give an ability to contribute to the sample.

Reply:
@vinokurig Of course, as the registry repo open a PR with any desired changes to the starter project repo and we can review them.

Reply:
reverted the item